requirements.txt

```diff
@@ -32,3 +33,5 @@ pyinotify>=0.9.5,<=0.10
 # used by nmap actions
 python-nmap>=0.3.4,<0.4
 semver>=2.1.2
+# flat_file auth backend is installed and available by default
+git+https://github.com/StackStorm/st2-auth-backend-flat-file.git@master#egg=st2auth_flat_file_backend
```

Comment:
Hmm, I think we should look at packaging that and publishing to pypi. Is that something you already considered?